config/conf.json

```diff
@@ -104,7 +104,7 @@
             "packages" : {
                 
             },
-            "version" : "1.0.8",
+            "version" : "1.0.9",
```

Comment:
ah, thanks for updating the script..I think you need terra-docker-r under https://github.com/DataBiosphere/terra-docker/blob/a4bf66761880f95b0453a89c8bee682c92b29316/updateVersions.sc#L76 as well